service/src/main/java/org/jboss/aerogear/unifiedpush/service/impl/UserTenantInfo.java

```diff
@@ -0,0 +1,63 @@
+package org.jboss.aerogear.unifiedpush.service.impl;
+
+import java.util.Objects;
+import java.util.UUID;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class UserTenantInfo {
+
+	private final UUID userGuid;
+	private final UUID pushId;
+	private final String client;
+
+	public UserTenantInfo(UUID userGuid, UUID pushId, String client) {
+		this.userGuid = userGuid;
+		this.pushId = pushId;
+		this.client = client;
+	}
+
+	@JsonProperty("u")
+	public UUID getUserGuid() {
+		return userGuid;
+	}
+
+	@JsonProperty("p")
+	public UUID getPushId() {
+		return pushId;
+	}
+
+	@JsonProperty("c")
+	public String getClient() {
+		return client;
+	}
+
```

Comment:
Are we limited somehow to extremely short properties (1 letter) ? If we do need to save space can we at least use 3 letters - `uid`, `pid`, `cid` - so that next property we add won't cause us problems. E.g. it we add `permissions` - we cannot use `p` so we will probably use some non-intuitive letter. However, if we use 3 letters we could use `prm`.